src/routes/AppRouter.tsx

```diff
@@ -2,6 +2,7 @@ import React, { Suspense } from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import Layout from "../components/Layout";
 import PrivateRoute from "./PrivateRoute";
+import index from "../components/HomeScreen";
```

Comment:
aca podes importar el componente al igual que el otro.
```suggestion
const HomeScreen= React.lazy(() => import("../components/HomeScreen"));
```